bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/SimpleRow.java

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.grpc.scanner;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.List;
+
+import com.google.appengine.repackaged.com.google.common.base.Objects;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Ordering;
+import com.google.protobuf.ByteString;
+
+/**
+ * <p>This class stores values from {@link com.google.bigtable.v2.ReadRowsResponse} objects and
+ * represents a single row.</p>
+ *
+ * @author tyagihas
+ * @version $Id: $Id
+ */
+public class SimpleRow implements Serializable {
+
+  private static final long serialVersionUID = 1L;
+
+  public final static class SimpleCell implements Serializable {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Ok, makes sense. Only concern I have is whether we (or Dataflow) has a test to make sure that we do/don't maintain serialization compatibility for whatever boundary is crossed there, e.g. if we rework this class/change/remove/add fields or whatever. Maybe it doesn't matter because the versions will always match?